### PR TITLE
envy's knife is now called envy's blade, and it has a new sprite to reflect

### PR DESCRIPTION
**Pull request by ghost**
the knife was just a ritual knife, and using old skins is so bad it should be met with a bonk on the head.

(The cooldown between transformations takes awhile, so be patient!)

### CODE REVIEW

File: code/modules/ruins/objects_and_mobs/sin_ruins.dm [hunk 1]
```diff
@@ -110,7 +110,7 @@
 	name = "envy's knife"
 	desc = "Their success will be yours."
 	icon = 'icons/obj/wizard.dmi'
-	icon_state = "render"
+	icon_state = "envy"
 	item_state = "knife"
```
Comment (ShizCalev) [inline code/modules/ruins/objects_and_mobs/sin_ruins.dm]: inhand is still a kitchen knife 🤔

Comment (Armhulen) [reply code/modules/ruins/objects_and_mobs/sin_ruins.dm]: how am i supposed to make inhands for this

Comment (optimumtact) [reply code/modules/ruins/objects_and_mobs/sin_ruins.dm]: by getting good

**Comment by Armhulen** [comment]
op sprite is outdated but i added red eyes and fixed the weird pitchfork transform

**Comment by Firecage** [comment]
Mmm, it has a grip though, so I assume this isn't just the blade part itself? Perhaps you shouldn't name it a blade.

**Comment by Jordie0608** [comment]
The sprite looks like a kitchen knife sharpener.
index

**Comment by Armhulen** [comment]
@Firecage suggestons
@Jordie0608 safe space

**Comment by Bawhoppen** [comment]
I envy a better sprite

**Comment by Armhulen** [comment]
better than copy paste i'll leave it at that

**Comment by Bawhoppen** [comment]
I just wanted to make the joke actually I don't really care

**Comment by Armhulen** [comment]
ok :)

**Comment by WJohn** [comment]
Mmm. This needs a better sprite. The first one does indeed look like a kitchen sharpener or something to that effect, and the trident is terribly shaded and flat. It is however cool how it changes from weapon to weapon, so I'm fine with that staying around after a resprite.

**Comment by Armhulen** [comment]
OK, i'll try better I suppose

**Comment by Jordie0608** [comment]
@Armhulen Here, use this as a reference model

**Comment by Armhulen** [comment]
1. nothing wrong with me
2. nothing wrong with me
3. nothing wrong with me
4. nothing wrong with me

**Comment by optimumtact** [comment]
1. Something's got to give
2. Something's got to give
3. Something's got to give now

**Comment by Armhulen** [comment]
what

**Comment by JJRcop** [comment]
I edited the timings of the gif so it's easier.
```https://user-images.githubusercontent.com/5211576/31694006-af3cf828-b36f-11e7-9149-4b3ce4039943.gif```

**Comment by Armhulen** [comment]
I'm gonna get around to redoing the sprite, hol up

**Comment by Armhulen** [comment]
i really can't bother at the moment actually

**Pull request closed without merging**